zarr/util.py

```diff
@@ -234,17 +234,8 @@ def is_total_slice(item, shape: Tuple[int]) -> bool:
     if isinstance(item, tuple):
         return all(
             (
-                (
```

Comment:
Alternatively I could comment this out and add a TODO

Reply:
From my side, +1 for reverting. We can also revert the revert.